packages/server/src/routes/api/users/users-router.js

```diff
@@ -59,7 +64,22 @@ router.post(
   async (req, res, next) => {
     try {
       const validationResults = await validateEmailAndUsername(req.body);
-      res.status(200).send({ ...validationResults, emailExists: false });
+      res.status(200).send({ ...validationResults });
+    } catch (err) {
+      next(err);
+    }
+  },
+);
+
+router.patch(
+  "/:id",
```

Comment:
Should probably just get id from the token

Reply:
change from #26